elements/pfe-tooltip/test/pfe-tooltip.spec.ts

```diff
@@ -0,0 +1,129 @@
+          </path>
+        </svg>
+        <div slot="content">
+          Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna
+          aliqua. Mi eget mauris pharetra et ultrices.
+        </div>
+      </pfe-tooltip>
+    </div>
+  `;
+};
+
+describe('<pfe-tooltip>', function() {
+  beforeEach(async function() {
+    await setViewport({ width: 1000, height: 1000 });
+  });
+  it('should upgrade', async function() {
+    const element = await fixture<HTMLDivElement>(blankElement);
+    const klass = customElements.get('pfe-tooltip');
+    expect(element)
+      .to.be.an.instanceOf(klass)
+      .and
+      .to.be.an.instanceOf(PfeTooltip);
+  });
+
+  it('should be accessible', async function() {
+    const element = await fixture<HTMLDivElement>(basicElement);
+
+    expect(element).shadowDom.to.be.accessible();
+  });
+
+  it('should reflect left position', async function() {
+    const element = await fixture<HTMLDivElement>(createElement('left'));
+
+    const placement = element?.querySelector('pfe-tooltip')?.shadowRoot?.querySelector('[id^=pfe-tooltip]')?.getAttribute('data-popper-placement');
+
+    expect(placement).to.equal('left');
+  });
+
+  it('should reflect top position', async function() {
+    const element = await fixture<HTMLDivElement>(createElement('top'));
+
+    const placement = element?.querySelector('pfe-tooltip')?.shadowRoot?.querySelector('[id^=pfe-tooltip]')?.getAttribute('data-popper-placement');
+
+    expect(placement).to.equal('top');
+  });
+
+  it('should reflect right position', async function() {
+    const element = await fixture<HTMLDivElement>(createElement('right'));
+
+    const placement = element?.querySelector('pfe-tooltip')?.shadowRoot?.querySelector('[id^=pfe-tooltip]')?.getAttribute('data-popper-placement');
+
+    expect(placement).to.equal('right');
+  });
+
+  it('should reflect bottom position', async function() {
+    const element = await fixture<HTMLDivElement>(createElement('bottom'));
+
+    const placement = element?.querySelector('pfe-tooltip')?.shadowRoot?.querySelector('[id^=pfe-tooltip]')?.getAttribute('data-popper-placement');
+
+    expect(placement).to.equal('bottom');
+  });
+
+  it('should reflect offset position', async function() {
+    const element = await fixture<HTMLDivElement>(createElement('bottom', [100, 100]));
+
+    const pfeTooltipElement = element?.querySelector('pfe-tooltip');
+
+    pfeTooltipElement?.click();
+
+    await pfeTooltipElement?.updateComplete;
+
+    const style = element?.querySelector('pfe-tooltip')?.shadowRoot?.querySelector('[id^=pfe-tooltip]')?.getAttribute('style');
+
+    expect(style).to.exist;
+  });
+
```

Comment:
Consider adding tests for other position variants `left-start, left-end, right-start, right-end etc`